gui/src/main.rs

```diff
@@ -234,13 +234,12 @@ fn run_launcher(
     drop(win);
 
     // Extract GUI states.
-    let start = Rc::into_inner(start).unwrap().into_inner();
+    let start = Rc::into_inner(start)
+        .unwrap()
+        .into_inner()
+        .then_some(VmmArgs {});
```

Comment:
This one should not construct here because we need to pull the other GUI states to construct it (e.g. debug address).

Reply:
Ok, but we can refactor that later, this is equivalent to what's in the code now, just more 'Rusty'

Reply:
On the other hand, this is a simple fi